packages/browser-utils/src/metrics/browserMetrics.ts

```diff
@@ -539,6 +540,10 @@ export function _addResourceSpans(
   setResourceEntrySizeData(attributes, entry, 'encodedBodySize', 'http.response_content_length');
   setResourceEntrySizeData(attributes, entry, 'decodedBodySize', 'http.decoded_response_content_length');
 
+  if (entry.deliveryType != null) {
+    attributes['http.response_delivery_type'] = entry.deliveryType || 'default';
```

Comment:
I'm falling back to `"default"` here because it seems like we drop empty strings (which is the actual default of the property). Ideally we wouldn't need to do this but I don't see a better way at the moment.

Reply:
Maybe add an explaining comment to the code why this is needed